packages/synapse-react-client/src/components/ExperimentalMode/ExperimentalMode.tsx

```diff
@@ -35,16 +35,15 @@ const ExperimentalMode: React.FunctionComponent<ExperimentalModeProps> = ({
   }, [])
 
   const createExperimentalModeCookie = () => {
-    cookies.set(EXPERIMENTAL_MODE_COOKIE, { path: '/' })
+    cookies.set(EXPERIMENTAL_MODE_COOKIE, 'true', { path: '/' })
```

Comment:
The second parameter is the cookie value, the third parameter contain the options